css/styles.css

```diff
@@ -321,3 +402,290 @@ textarea {
   left: 4px;
   font-size: 11px;
 }
+
+.desktopnav {
+  visibility: hidden;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: lightgray;
+}
+
+.desktopnav ul li {
+  margin-right: 30px;
+}
+
+.awesomedeco1 {
+  background-image: url(../resources/image-geometry_7.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 1488px;
+  top: 296px;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+}
+
+.awesomedeco2 {
+  background-image: url(../resources/image-geometry_6.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+  left: -86px;
+}
+
+.imggeo2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imggeo5 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.fixGap1 {
+  margin-bottom: -5px;
+}
+
+.imgGroup83 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.redSquare {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imageDots2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+@media screen and (min-width: 1680px) {
+  .decoration1 {
+    visibility: hidden;
+  }
+
+  .decoration2 {
+    visibility: hidden;
+  }
+
+  .decoration3 {
+    background-image: url(../resources/Thur.png);
+    background-repeat: no-repeat;
+    padding-bottom: 164px;
+    height: 877.09px;
+    visibility: visible;
+    width: 1403px;
+    margin-left: 677px;
+    position: relative;
+  }
+
+  .desktopflex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .desktopnav {
+    visibility: visible;
+  }
```

Comment:
- [ ] Kindly make the navbar centered as shown in Figma

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187050206-d05cc0fe-d877-4a0e-8aa5-f1809eec739f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187050216-7dd8d0cf-670d-47fb-a9e0-2b8a4af6f9cd.png)